stacks/dovetail.prx.org.yml

```diff
@@ -510,6 +510,53 @@ Resources:
           Value: !Ref AWS::StackName
         - Key: "prx:cloudformation:stack-id"
           Value: !Ref AWS::StackId
+  # Dovetail Elixir Alarms
+  DovetailRouterLogErrorsMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      FilterPattern: '{ $.level = "error" }'
+      LogGroupName: !Ref DovetailRouterLogGroup
+      MetricTransformations:
+        - MetricName: !Sub "dovetail_router_errors_${EnvironmentTypeAbbreviation}"
+          MetricNamespace: LogMetrics
+          MetricValue: "1"
+  DovetailRouterLogErrorsAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      AlarmName: !Sub "[DovetailRouter][LogErrors] ${EnvironmentType}"
+      AlarmActions: [!Ref OpsErrorMessagesSnsTopicArn]
```

Comment:
We don't really do inline arrays in our templates. We could start I suppose… but it's non standard as things stand.